tests/010-test-use-devs-to-create-docker-root-volume.sh

```diff
@@ -19,6 +19,7 @@ test_docker_root_volume() {
 DEVS="$devs"
 VG=$vg_name
 DOCKER_ROOT_VOLUME=yes
+DOCKER_ROOT_VOLUME_SIZE=40%FREE
```

Comment:
I see, you have removed `DOCKER_ROOT_VOLUME_SIZE=40%FREE` from the .conf file. So it won't be picked up by default. so that means if the user wants to use `DOCKER_ROOT_VOLUME` he has to explicitly pass both these options.